service/build.gradle

```diff
@@ -109,6 +109,10 @@ dependencies {
     testImplementation 'org.awaitility:awaitility'
     testImplementation 'org.testcontainers:postgresql:1.17.3'
 
+    // security-related dependency upgrades of transitive dependencies
+    implementation 'org.yaml:snakeyaml:2.0'
```

Comment:
just OoC, is this section pretty much locking in versions of certain dependencies?  if so would it be possible to put them next to the thing pulling it with a "default" version or is it kind of all over the place?

Reply:
good point, they're all inherited through tcl - see if you think the reorg makes more sense